/**:
  ros__parameters:
    control_command: "raw"
    recordreplay_command_enabled: false
    # Which index in the joy message corresponds to which axis
    axes:
      # In our version of the joystick driver, axes typically go from -1.0 to 1.0
      # Triggers go from 1.0 (when in neutral position) to -1.0 (when fully squeezed)
      brake: 2  # Left trigger
      throttle: 5  # Right trigger
      front_steer: 0  # Left joystick, x direction
      rear_steer: 3  # Right joystick, x direction
      # curvature:
      acceleration: 1  # Left joystick, y direction
      # velocity:
    # Scale and offset of axes: val = (raw * scale) + offset
    axis_scale:
      brake: -50.0
      throttle: -50.0
      front_steer: 100.0
      rear_steer: 100.0
      # curvature:
      # acceleration:
      # velocity:
    axis_offset:
      brake: 50.0
      throttle: 50.0
      front_steer: 0.0
      rear_steer: 0.0
      # curvature:
      # acceleration:
      # velocity:
    # Buttons are not yet implemented
    # Index for which button is which
    buttons:
      # autonomous:
      headlights: 2  # X
      # wiper:
      gear_drive: 0  # A
      gear_reverse: 3  # Y
      gear_park: 1  # B
      # gear_neutral:
      # gear_low:
      # blinker_left:
      # blinker_right:
      # blinker_hazard:
      # velocity_up:
      # velocity_down:
      recordreplay_start_record : 7 # start button
      recordreplay_start_replay : 8 # logo button
      recordreplay_stop : 6 # back button
